ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/_cq_editConfig.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
     jcr:primaryType="cq:EditConfig"
-          cq:actions="[edit]"
+          cq:actions="[edit,delete]"
```

Comment:
this should also be supported after forms+sites

Reply:
added this since in test we do insert and then delete. If this will be going later, we will only try adding and don't delete this component since it's not available.